### DISCUSSION TITLE
SIMWE model: R.SIM.WATER

### DISCUSSION THREAD

**Discussion by xycarto** [OSGeo/grass#4475] [Q&A] [answered]
Maybe the GRASS discussion board is not the best for this, but I am having some significant difficulty finding any discussion of the `r.sim.water` tool and the SIMWE model it uses. Maybe someone here could help point me in the right direction?

By far, the `r.sim.water` tool is one of the best tools for flood modelling in terms of scalability and development into scripted methods. 
I have been doing a good amount of research on the tool you developed and have a few questions if anyone has the time.  I am very interested in developing this tool into a rapid flood prediction model at a national scale.  I have found there is not a lot of information about the SIMWE process out there, outside of a few academic papers that implemented the model in urban spaces but those papers do not talk much about the SIMWE process itself.

Here are some questions I am trying to find answers to:

1. The biggest difficulty I am facing is trying to convince others this is a good tool for a generalized flood prediction model. There seems to be little documentation to back up the use of this tool or its implementation. Many folks I talk to would like me to move to either a SWAT or HEC-RAS approach to flood modelling.  I like this tool, as it is a perfect set up for scale dprocessing, but I get the feeling even the authors may not be using it much these days.  My question is, is the SIMWE model outdated in favour of newer models or is it still relevant today?  If so, is there some sort of documentation I use to demonstrate this?
2.  I do have some difficulties in developing perimeters for the tool, since there is little explanation describing what the purpose of each perimeter is. Is there a definitive guide that an engineer like myself might be able to use that better describes the function of the perimeters?
3. Overall, I am following a lost cause for the SIMWE model and should I focus my attention solely on SWAT or HEC-RAS?

**Comment by petrasovaa** [comment, marked as answer]
Hi,

SIMWE is not outdated and it is still being used, but I agree the documentation could be better. Here is lecture, maybe you can find some useful info: https://www.youtube.com/watch?v=v0jj4UKJmoQ&list=PLiNYVMuYqTHXjIZyTSpiNzCAADzeabNlc&index=31

But academic papers should have the parametrization of the model described, so I would assume you could find useful information there.

**Comment by xycarto** [comment]
Cheers @petrasovaa 

The presentation is quite helpful and will get me a bit further. It helps with some of the perimeterization of the tool. Thank you for the response :) 

It at least drives me to take a closer look at my elevation model. I am just not getting the flow I would expect from the tool, but this is very likely user error and not the tool.

I will continue to look around for further explanation of the perimeters. Ideally, if I could just talk to an expert for about 20 minutes, that'd be ace.  In the meantime, I will see if I can get an example to share around up an running on Git.

**Comment by petrasovaa** [comment]
The model is for shallow water flow, so it's possible you have issues in some areas, because they are not "shallow"... In these cases you can try increase diffusion by lowering hmax or increasing halpha.

**Comment by xycarto** [comment]
Awesome!  Thanks @petrasovaa !

I think this is what I was looking for "_try increase diffusion by lowering hmax or increasing halpha_"

**Comment by metzm** [comment]
Another question about `r.sim.water`:

The meaning of the dimensionless Manning’s coefficient (options `man` and `man_value`) is not clear. Apparently it is an indicator for surface roughness, but the meaning of the values is nowhere documented.

- Higher values mean smoother surfaces?
- The allowed range is ?
- According to the formula in the references, it seems that a value of 0 (zero) is not allowed because it would cause division by zero?

In [http://fatra.cnr.ncsu.edu/~hmitaso/gmslab/papers/LLEmiterev1.pdf](http://fatra.cnr.ncsu.edu/~hmitaso/gmslab/papers/LLEmiterev1.pdf) (a reference listed in the manual) there is a section "4.3.2 Grassed waterways." that illustrates the effects of changes in Manning's coeffcient, "topographic conditions and roughness within the grassed area, represented by Manning's coefficient".  But this section does not answer the above questions.

A clarification on the meaning and allowed range of the values of Manning's coefficient would be very helpful!

**Reply by marisn** [reply]
Here's some read: [Fischenich 2000. Robert Manning (A Historical Perspective) ](https://erdc-library.erdc.dren.mil/server/api/core/bitstreams/de30b63c-2d7a-4816-8902-337e818e9548/content)

**Reply by metzm** [reply]
The part relevant to `r.sim.water` seems to be "n is the Manning resistance coefficient"

It seems that higher values mean rougher surfaces and a value of zero is not allowed.
Still open question, particularly with regard to [http://fatra.cnr.ncsu.edu/~hmitaso/gmslab/papers/LLEmiterev1.pdf](http://fatra.cnr.ncsu.edu/~hmitaso/gmslab/papers/LLEmiterev1.pdf):
- The allowed/reasonable range is ?

**Reply by neteler** [reply]
While I don't know the range answer, this tutorial by @baharmon might be of interest (it contains a Manning table):

https://baharmon.github.io/hydrology-in-grass

**Reply by baharmon** [reply]
A couple more resources. In the GRASS wiki, there is a table for deriving [Manning's n values from NLCD](https://grasswiki.osgeo.org/wiki/NLCD_Land_Cover#Manning's_n). There is also a paper on deriving Manning's n values from the global [LULC 2020-ESRI](https://www.mdpi.com/2073-4441/14/23/3963) dataset.

**Reply by neteler** [reply]
... totally worth to add this/the URLs to the `r.sim.water` manual page :-)

**Reply by metzm** [reply]
Thanks a lot for these helpful resources! I will create a PR updating the manual and then this discussion once the PR has been approved.

**Reply by neteler** [reply]
(this PR has been merged and also been backported to GRASS GIS 8.4.)

**Comment by 13greensky** [comment]
Hello, I just started using SIMWE and I encountered some problems when running "r.sim.water", which caused the results to be wrong.

At first, I could get a relatively correct water_depth map by only inputting a rain_value (such as 50mm/h) and Manning's raster map, setting nwalkers to 10000, and keeping the rest as default values.

But I couldn't get the correct water_depth when I replaced "rain_value" with a "rain's map" based on "infil's map" of different land use types (where I set the infiltration rate of water and built-up land to 0).

In the wrong simulation results, only the parts with 0 infiltration rate have water_depth, and all other plots are 0.

After referring to your post, I adjusted "niterations" to 240min, "output_step" to 10min, "diffusion_coeff" value unchanged, "hmax" value reduced to 0.1, and "halpha" increased to 6, but the results still did not change.

Hope to get your help.

**Comment by petrasovaa** [comment]
Could you provide something reproducible, I would at least need to know the infiltration values you are using.

**Reply by 13greensky** [reply]
Of course!!

The 5 pictures in Figure 1 are my process and results:
(a) Input only fixed rain_value: 100;
(b) Water_depth calculation result of (a) based on (e) parameter
(c) Input rainfall excess rate and runoff infilration rate map, the value is shown in Figure 2
(d) Water_depth calculation result of (c) based on (e) parameter
(e) parameters setting

Figure 2 is rainfall excess rate and runoff infilration rate map.
Detailed infiltration values ​​(mm/h) are as follows:
Grassland: 25.8
Forest: 45.6
Farmland: 21.6
Bare land: 4.8
Building: 0.00
Water: 0.00
![figure1](https://github.com/user-attachments/assets/d22c7b36-ce70-49b3-9678-1977f3acd1df)
![figure2](https://github.com/user-attachments/assets/08a3d5d2-929d-43f7-9715-7a345e456516)

**Reply by joyceleppan7-afk** [reply]
olá,! Você consegui utilizar o simwe e fazer a calibração do modelo?
